SimplioOfficial/simplio-app
# Task/141 update network wallet balance

# Overview

Updating balance of a user's NetworkWallet. Wallet Repository exposes methods to refresh balances for AccountWallet, AssetWallet and NetworkWallet. Account update lifetime is 180s by default.

## Test cases

1. Add an asset to inventory
2. Go to receive screen
3. Send resources to the wallet
4. Go to Inventory. Balance on network wallet should be updated. Remember balance is updated every 3 minutes. So enter to inventory after 3 minutes passed.
